resource/bem/index.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html>
+
+<head>
+    <title> Prechu </title>
+    <link rel="stylesheet" type="text/css" href="../bem/css.css" />
+</head>
+
+<body>
+    <div class="page">
+        <div class="page__header">
+            <div class="multi-columns">
+                <div class="page__header__left">
+                    <div class="branch">
+                        <img src="../images/logo.png" alt="" class="branch__icon" />
+                        <span class="branch__name"> Prechu </span>
+                    </div>
+                </div>
+                <div class="page__header__right">
```

Comment:
it seems we have problem with two-columns component. Please contact others for help